server/dive_tasks/tasks.py

```diff
@@ -415,7 +412,14 @@ def convert_video(self: Task, path, folderId, auxiliaryFolderId, itemId):
     if process.returncode == 0:
         manager.updateStatus(JobStatus.PUSHING_OUTPUT)
         new_file = gc.uploadFileToFolder(folderId, output_path)
-        gc.addMetadataToItem(new_file['itemId'], {"codec": "h264"})
+        gc.addMetadataToItem(
+            new_file['itemId'],
+            {
+                "source_video": False,
+                "transcoder": "ffmpeg",
```

Comment:
Just a question:
Is `transcoder: "ffmpeg"` there with the assumption that we may use the VIAME stuff in the future for image/color enhancement?

Reply:
Yes.  `source_video=False` was needed as a difinitive marker that this is a derived video, not source.  `transcoder` indicates what did the conversion.   `codec` just has the codec, which shouldn't be used as a proxy for any other properties as it was before.